update_prevalence.py

```diff
@@ -1582,21 +1589,21 @@ def format_obj(obj: Dict[str, Any], indent: str = "  ") -> None:
     (csvdir / "date.csv").write_text("Date\n{}".format(effective_date.strftime("%Y-%m-%d")))
     with (csvdir / "index.csv").open("w") as topfile:
         topfile.write("Location,Slug\n")
-        for slug, data in app_locations.items():
-            if not data.topLevelGroup:
+        for slug, location_date in app_locations.items():
+            if not location_date.topLevelGroup:
```

Comment:
The variable `data` was reused here; mypy doesn't like having the type redefined on an existing variable.